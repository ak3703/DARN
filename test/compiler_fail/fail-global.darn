int b;
int a;
int b; /* duplicate global */

int main() {
	return 0;
}
