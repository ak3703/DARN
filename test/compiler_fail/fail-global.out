Fatal error: exception Failure("Duplicate global b")
